app/controllers/application_controller.rb

```diff
@@ -114,7 +114,7 @@ def after_accept_path_for(_resource)
   end
 
   def raise_suspended
-    raise "SUSPENDED" if current_user&.banned
+    raise "SUSPENDED" if current_user&.suspended
```

Comment:
I know you just changed the method name here, but maybe now is a good time to move this from a `RuntimeError` with a message of `"SUSPENDED"` to an actual error (see `app/errors`), like e.g. 

```ruby
# app/errors/suspended_error.rb
class SuspendedError < StandardError; end
```